### DISCUSSION TITLE
I extracted Wifi icons from macOS Big Sur

### DISCUSSION THREAD

**Discussion by Human79** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#53] [Ideas]
I have extracted wifi icons from Big Sur in .png form, so is it possible to use them? If not, I can covert them to .icns. Please keep this open as I am finding a to enable auto dark/light icons. Do you have any idea how apple changes wifi icon according to background? What is the resolution of the icons? Attaching wifi icons.
[WiFi Icons.zip](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/files/6616912/WiFi.Icons.zip)
Sorry for both opening an issue and writing here.

**Comment by chris1111** [comment]
It is not possible with StatusBarApp.app because Dark and Light mode is not support that's why I integrate a Blue Icon

**Reply by Human79** [reply]
Yeah, noticed it . By the way, do you know how Apple manages to change it according to the background?

**Comment by chris1111** [comment]
No its build with the Finder
